multimedia/ffmpeg6/Portfile

```diff
@@ -539,6 +544,20 @@ This build of ${name} includes no GPLed or nonfree code and is therefore license
 "
 }
 
+notes-append "
+--------
```

Comment:
Please remove ASCII art. If notes should have dividers, MacPorts base should add them, not each individual port.